src/Microsoft.IdentityModel.Tokens/CryptoProviderFactory.cs

```diff
@@ -103,7 +103,9 @@ public CryptoProviderFactory(CryptoProviderFactory other)
         /// <summary>
         /// Gets the <see cref="CryptoProviderCache"/>
         /// </summary>
-        public CryptoProviderCache CryptoProviderCache { get; } = new InMemoryCryptoProviderCache();
+        /// Set was added for simplifying stress and perf tests.
+        /// We may want to remove it.
```

Comment:
>/// We may want to remove it. [](start = 8, length = 29)

want to make it internal then?

Reply:
@henrik-me i think internal makes good sense could be helpful for testing

Reply:
Made the change here: https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/1605/commits/9c7858b65c91a66f869a97950ee007dc282bd6fb